usps-processor@.service

```diff
@@ -0,0 +1,40 @@
+[Unit]
+Description=USPS Processor
+
+After=docker.service
+Requires=docker.service
+After=consul@%i.service
+Wants=consul@%i.service
+After=wildfly@%i.service
+Requires=wildfly@%i.service
+
+[Service]
+EnvironmentFile=/etc/environment
+TimeoutStartSec=10min
+TimeoutStopSec=10min
+
+Environment=DOCKER_REPO=quay.io/democracyworks/usps-processor
+Environment=VERSION=buildkite-coreos-fd55ab2
```

Comment:
For now, how do you feel about putting `latest` as the repo-committed value of this? We need to figure out a versioning system for our components.